### DISCUSSION TITLE
save segmentation results back to label version

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#3623] [Q&A] [answered]
Hi，Thanks for the excellent work. I am using MONAI for multi-class 3D medical image segmentation task.
My task has 27 classes(including background). When training I use "DiceCELoss(include_background=True,to_onehot_y=True,softmax=True, sigmoid=False)"  to convert 1 channel label files to 27 channels. 
When training finished, how to convert the 27channels prediction back to 1channel style?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @yingdiLee ,

Thanks for your interest here.
You can take a look at the `AsDiscrete` postprocessing transform:
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/torch/unet_evaluation_dict.py#L63
You can set `argmax` for your case I think.

Thanks.
